schema.sql

```diff
@@ -0,0 +1,10 @@
+postgres=# CREATE DATABASE vet_clinic;
+postgres=# \c vet_clinic
+
+vet_clinic=# CREATE TABLE animals (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(25),
+  date_of_birth TIMESTAMP,
+  escape_attempts INT,
+  neutered BOOLEAN,
+  weight_kg DECIMAL);
```

Comment:
- [ ] Kindly remove **postgres=#** and **vet_clinic=#** cuz these are the words which your terminal has. But they are not part of your query so kindly remove them :blush: